dev-packages/browser-integration-tests/suites/tracing/trace-lifetime/navigation/test.ts

```diff
@@ -119,7 +119,7 @@ sentryTest('error after navigation has navigation traceId', async ({ getLocalTes
   await page.locator('#errorBtn').click();
   const [errorEvent, errorTraceHeader] = await errorEventPromise;
 
-  expect(errorEvent.type).toEqual(undefined);
+  expect(errorEvent.type).toEqual('event');
```

Comment:
h: let's not change this please unless there's a good reason to (is there?). So far, sending error events without a type worked well.